tapioca/tapioca.py

```diff
@@ -192,26 +192,31 @@ def _make_request(self, request_method, *args, **kwargs):
             data = self._api.process_response(response)
         except ResponseProcessException as e:
             client = self._wrap_in_tapioca(e.data, response=response,
-                                           request_kwargs=request_kwargs)
-            raise e.tapioca_exception(client=client)
+                                            request_kwargs=request_kwargs)
+            tapioca_exception = e.tapioca_exception(client=client)
+            if refresh_auth and self._api.is_authentication_expired(tapioca_exception):
+                self._api.refresh_authentication()
+                return self._make_request(request_method, refresh_auth, args, kwargs)
+            else:
+                raise tapioca_exception
 
         return self._wrap_in_tapioca(data, response=response,
                                      request_kwargs=request_kwargs)
 
-    def get(self, *args, **kwargs):
-        return self._make_request('GET', *args, **kwargs)
+    def get(self, refresh_auth=False, *args, **kwargs):
```

Comment:
Just realized, no need to pass ``refresh_auth`` in none of this methods since it goes straight to ``_make_request`` and kwargs are being passed as well.